components/org.wso2.carbon.identity.query.saml/.gitignore

```diff
@@ -0,0 +1,22 @@
+*.class
```

Comment:
this file is not needed since we already have a .gitignore file for identity-inbound-auth-saml repo.